checker.cs

```diff
@@ -3,32 +3,81 @@
 
 class Checker
 {
-    static bool vitalsAreOk(float bpm, float spo2, float respRate) {
-        if(bpm < 70 || bpm > 150) {
-            return false;
-        } else if(spo2 < 90) {
-            return false;
-        } else if(respRate < 30 || respRate > 95) {
-            return false;
+    public static string isBPMok(float bpm)
+    {
```

Comment:
All the is...ok functions are similar with small changes in them. Resolve the duplication with a range-checker.